src/frontend/src/components/common/Modal/index.tsx

```diff
@@ -18,7 +19,7 @@ const Modal = ({
         <span className="modal-title"> {titleText}</span>
         <div className="modal-horizontal-divider"></div>
         {children}
-        <div className="modalbtn__container">
+        <div className={`modalbtn__container ${reversed ? 'reversed' : ''}`}>
```

Comment:
IF there is only one calling function, can we default to reversed? IF not, ignore this message.

Reply:
I see it being called again in PIAIntakeForm